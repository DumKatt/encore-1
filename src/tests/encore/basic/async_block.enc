def square(x: int): int
  x * x

class Main
  def main(): void {
    repeat i <- 5
      async {
        let s = square(i) in
          print square(s)
      }
  }
