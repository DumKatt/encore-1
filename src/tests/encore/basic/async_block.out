0
1
16
81
256
